src/Countries/Bulgaria.php

```diff
@@ -0,0 +1,82 @@
+<?php
+
+namespace Spatie\Holidays\Countries;
+
+use Carbon\CarbonImmutable;
+
+class Bulgaria extends Country
+{
+    public function countryCode(): string
+    {
+        return 'bg';
+    }
+
+    /**
+     * @param int $year
+     * @return array<string, string>
+     */
+    protected function allHolidays(int $year): array
+    {
+        return array_merge([
+            'Нова година' => '01-01',
+            'Ден на Освобождението на България от османско робство' => '03-03',
+            'Ден на труда и на международната работническа солидарност' => '05-01',
+            'Великден – християнската религия отбелязва Възкресение Христово' => '05-05',
+            'Гергьовден, Ден на храбростта и Българската армия' => '05-06',
+            'Денят на българската просвета и култура и на славянската писменост' => '05-24',
+            'Съединението на България' => '09-06',
+            'Ден на независимостта на България' => '09-22',
+            'Коледа ден 1' => '12-24',
+            'Коледа ден 2' => '12-25',
+            'Коледа ден 3' => '12-26',
+        ], $this->variableHolidays($year));
+    }
+
+    /**
+     * @param int $year
+     * @return array<string, string>
+     */
+    protected function variableHolidays(int $year): array
+    {
+        $easter = CarbonImmutable::createFromTimestamp($this->calculateEasterDate($year))
+            ->setTimezone('Europe/Sofia');
+
+        return [
+            'Великден – християнската религия отбелязва Възкресение Христово' => $easter->format('m-d'),
+        ];
+    }
+
+    /**
+     * Calculate the date of Easter using the Computus algorithm with Carbon
+     * @param int $year
+     * @return CarbonImmutable
+     * @throws \RuntimeException if the calculated date is invalid
+     */
+    protected function calculateEasterDate($year): CarbonImmutable
```

Comment:
Is this an orthodox calculation? We already have one in Country.php. Does that one work too?

Reply:
I totally didn't see that. Yes it works for Bulgaria too.

Reply:
I think you used the wrong one.

Reply:
Yep you're right.. Its correct for 2024 `5 May 2024`, but for 2025 it should be `20 April 2025`. I'm not sure which calculation to use..

Reply:
Ok using the `easter_date` function directly actually resolves the issue.

Reply:
Ok, nevermind.. 🤣  It's now wrong for 2024, im confused..

Reply:
Country.php has a easter() and orthodoxEaster() function. I suggest to use one of those. 

If the orthodox calculation gives a false result, feel free to update it, in order to make it work. Do verify if the results for NorthMacedonia is still correct.

Reply:
Neither functions give proper result, i will investigate more.

Reply:
The problem is that this function in Country.php is wrong calculating the OrthodoxEastern. Please check Greece #114 to fix the problem. @Nielsvanpach needs to update orthodoxEaster() 

> Country.php has a easter() and orthodoxEaster() function.

Reply:
It's still the wrong function you're calling. You should use orthodoxEaster()